app/boards/arm/kbdfans_tofu65_v2/kbdfans_tofu65_v2.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: kbdfans_tofu65_v2
+name: KBDfans Tofu65 2.0
+type: mcu
+arch: arm
+flash: 8192
+ram: 264
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - uart
+  - gpio
+  - adc
+  - i2c
+  - usb_device
+  - spi
+  - hwinfo
+  - watchdog
+  - pwm
```

Comment:
Should uart, adc, i2c, spi, and watchdog be removed from here since they are not being used? Or is this file meant to show the boards capabilities regardless of their use?

Reply:
You could. These files are an upstream Zephyr file we really don't leverage. Not critical either way.